spec/core_functions/color/complement.hrx

```diff
@@ -61,19 +82,51 @@ a {
   b: rgba(224, 64, 80, 0.7);
 }
 
+<===> alpha/output-libsass.css
+a {
+  b: rgba(224, 64, 80, 0.7);
+}
+
 <===>
 ================================================================================
-<===> named/input.scss
-a {b: complement($color: red)}
+<===> explicit_space/options.yml
+:ignore_for:
+  - libsass
 
-<===> named/output.css
+<===> explicit_space/input.scss
+a {b: complement(red, hwb)}
+
+<===> explicit_space/output.css
 a {
-  b: aqua;
+  b: hwb(180 0% 0%);
```

Comment:
Even if `complement()` uses another color space for its computation, its output is always in the original color space. We try to have an invariant that `color.to-space()` is the _only_ way to return a color in a different color space than it was passed in.

Reply:
That makes sense. These should all be returning in the same space as the original color now.